_includes/docs/latest/required_roles.md

```diff
@@ -1,27 +1,30 @@
-**Granted at the Organization level**
-
- `forseti_inventory` and `forseti_scanner` require:
+**Forseti server service account requires:
```

Comment:
This line was removed because it didn't make sense in the other places this include was used. I recommend you add this to the document you're including this one from.